com.tiny-beast-dictionary.yml

```
@@ -0,0 +1,27 @@
+app-id: org.Tiny-beast-dictionary
+runtime: org.gnome.Platform
+runtime-version: '42'
```

GNOME 42 runtime is EOL.

```suggestion
runtime-version: '44'
```